recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "cudatoolkit-dev" %}
 {% set release = "11.4" %}
-{% set version_build = "0" %}
+{% set version_build = "1" %}
```

Comment:
Hi jnooree - to bump the build number, you want to adjust the value at line 17 instead.

Else, If the intention is to build for a different version of cudatoolkit, then you will need to adjust additionally the driver versions for both platforms (lines 4 and 5). This way the correct files to retrieve can be resolved by the cudatoolkit-dev installer.

The right version numbers can be derived from the naming format of the runfile / executable. See [this](https://developer.nvidia.com/cuda-11-4-1-download-archive?target_os=Linux&target_arch=x86_64&Distribution=Ubuntu&target_version=20.04&target_type=runfile_local) for linux cudatoolkit 11.4.1 as example.

Reply:
Resolved by 32d9713. Thank you very much!